staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -30,40 +35,50 @@ function cleanup {
 }
 trap cleanup EXIT
 
-echo "Building client-gen"
-CLIENTGEN="${PWD}/client-gen-binary"
-go build -o "${CLIENTGEN}" ./cmd/libs/go2idl/client-gen
+function generate_group() {
+  local GROUP_NAME=$1
+  local VERSION=$2
+  local SERVER_BASE=${GOPATH}/src/${BASE_PATH}
+  local CLIENT_PKG=${BASE_PKG}/pkg/client
+  local LISTERS_PKG=${CLIENT_PKG}/listers_generated
+  local INFORMERS_PKG=${CLIENT_PKG}/informers_generated
 
-PREFIX=k8s.io/sample-apiserver/pkg/apis
-INPUT_BASE="--input-base ${PREFIX}"
-INPUT_APIS=(
-wardle/
-wardle/v1alpha1
-)
-INPUT="--input ${INPUT_APIS[@]}"
-CLIENTSET_PATH="--clientset-path k8s.io/sample-apiserver/pkg/client/clientset_generated"
+  echo "Building client-gen"
+  go build -o "${CLIENTGEN}" k8s.io/kubernetes/cmd/libs/go2idl/client-gen
 
-${CLIENTGEN} ${INPUT_BASE} ${INPUT} ${CLIENTSET_PATH} 
-${CLIENTGEN} --clientset-name="clientset" ${INPUT_BASE} --input wardle/v1alpha1 ${CLIENTSET_PATH} 
+  local PREFIX=${BASE_PKG}/pkg/apis
+  local INPUT_BASE="--input-base ${PREFIX}"
```

Comment:
Agree didn't like it either but it was the way it was written when I came to it, will update